frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx

```diff
@@ -52,14 +33,63 @@ export const MultilineUtilizationItem: React.FC<MultilineUtilizationItemProps> =
   }) => {
     const { t } = useTranslation();
     const { endDate, startDate, updateEndDate } = useUtilizationDuration();
+
+    const getCurrentData = (
+      description: string,
+      originalData?: DataPoint[],
+      dataUnitsValue?: string,
+    ): string => {
+      let current: string;
+      if (originalData && originalData.length > 0) {
+        const latestData = originalData[originalData.length - 1];
+        current = humanizeValue(latestData.y).string;
+        if (dataUnitsValue) {
+          current += ` ${dataUnitsValue}`;
+        }
+        if (description === 'in') {
+          current = t('console-shared~{{amount}} in', { amount: current });
```

Comment:
Shouldn't we be appending tis string to `current` ?

Reply:
We don't want to append because the placement of the "in" text could change based on the current language. It would be better to replace the text fully with a template string.